### DISCUSSION TITLE
Is there an official manual on the Marp project?

### DISCUSSION THREAD

**Discussion by szw0407** [marp-team/marp#407] [Q&A] [answered]
I am quite interested in the project and I wish to make full use of it. However, I'm not quite sure from where I can get a maual to learn about it. I searched Bing and found your official website, but cannot find a manual or getting-started instructions. I wish an official manual can be made so that users can use it better. Markdown, HTML and slides are quite different things. I am not quite sure how to do something like using a CSS in a markdown file, but just trying. If all features were listed, users would use it better.

**Comment by yhatt** [comment, marked as answer]
Please refer to #126. https://marp.app/docs is working in progress.

However, [each independent projects of Marp ecosystem](https://github.com/marp-team/marp#marp-family) are live actively, and the documentation on marp.app may not represent the current state. _The most reliable documents are README in each projects, or the project's website_ such as [Marpit framework](https://marpit.marp.app). These always present the current state of projects.
